extensions/prototype-content-with-issues-table/get_usage.R

```diff
@@ -45,14 +45,9 @@ get_usage_legacy <- function(client, from = NULL, to = NULL) {
 }
 
 get_usage <- function(client, from = NULL, to = NULL) {
-  tryCatch(
-    {
-      print("Trying firehose usage endpoint.")
-      get_usage_firehose(client, from, to)
-    },
-    error = function(e) {
-      print("Could not use firehose endpoint; trying legacy usage endpoints.")
-      get_usage_legacy(client, from, to)
-    }
-  )
+  from <- format(from, "%Y-%m-%dT%H:%M:%SZ")
+  to <- format(to, "%Y-%m-%dT%H:%M:%SZ")
+  # disable firehose for deploying example to connect.posit.it 
+  #get_usage_firehose(client, from, to)
+  get_usage_legacy(client, from, to)
```

Comment:
Can you explain the reasoning for why you're removing the `tryCatch` here?

Reply:
Since we removed the `tryCatch` around `get_jobs()` because connect.posit.it does not have malformed content the `tryCatch` here was also deleted, as it was checking for an endpoint we know is not available on connect.posit.it.